test/sepcomp/a/Makefile

```diff
@@ -35,12 +35,11 @@ include Makefile.include
 # But also:
 # - --cmi, for cross-module inlining, a must-have for anyone who relies on
 #   inline_for_extraction in the presence of interfaces
-# - --cache_checked_modules to rely on a pre-built ulib and krmllib
-# - --cache_dir, to avoid polluting our generated build artifacts outside o
+# - --cache_dir, to place checked files there
 
 FSTAR_EXE ?= fstar.exe
 FSTAR_NO_FLAGS = $(FSTAR_EXE) \
-  --odir obj --cache_checked_modules $(FSTAR_INCLUDES) --cmi \
+  --odir obj $(FSTAR_INCLUDES) --cmi \
```

Comment:
so cache_checked_modules is now the default?

Reply:
We always read checked files by default (this was gated by this option at some time long ago, but not now). This option only control whether to *write* the checked files, so I'm only using it (`-c`) in the rules to generate checked files. This is cleaner and less of a footgun IMO but happy to revert if you prefer.

Reply:
no this is great thanks for the explanation!